### DISCUSSION TITLE
How can I set 8bit encoding Laravel 9.x?

### DISCUSSION THREAD

**Discussion by hiteshsamcom** [symfony/symfony#47816] [Q&A]
### Symfony version(s) affected

6.0

### Description

How can I set the `8bit` encoder in notification?

Because in the `quoted-printable` encoder 3D tag is added to email logs and Gmail markup can't validate the schema.org.

```
return (new MailMessage)
            ->subject('8bit encoder subject)
            ->markdown('mails.booking', [
                'booking' => $this->booking
            ]);
```

### How to reproduce

It's produced into Laravel 9.x and Symfony 6.0

**Comment by xabbuh** [comment]
I am not sure I understand your issue. Can you elaborate a bit what’s happening and what you did expect instead?

**Reply by hiteshsamcom** [reply]
When I send an email it's added the 3D tag to the HTML content, it happened because of `Content-Transfer-Encoding: quoted-printable `so how can I change to `8bit` in laravel 9 because Swift is no longer for Laravel 9. and Symfony is not providing set encoder method.

https://user-images.githubusercontent.com/92365321/194800280-721b2408-634c-4af5-aaaf-1c49111c805c.png
